password-manager-start/password_generator.py

```diff
@@ -0,0 +1,28 @@
+from random import choice, shuffle, randint
+from tkinter import END
+
+LETTERS = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u',
```

Comment:
Put them in seperate file or a constant file, if they are to be used in multiple places, if not move them inside class. Better to keep class functionality in this class.

Same comment if for other class too.

Reply:
Refer latest commit